plugins/module_utils/version.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
 
 # Copyright: (c) 2021, Felix Fontein <felix@fontein.de>
```

Comment:
`reuse lint` does not like the colon after `# Copyright`. I would remove this to please it and to make this consistent with the other headers, but I'll leave it up to you :)